docs/source/design/clis.rst

```diff
@@ -49,6 +49,40 @@ Suppose you execute a script that defines 10 tasks and a workflow that calls onl
 
 It is considered fast registration because when a script is executed using ``pyflyte run``, the script is bundled up and uploaded to FlyteAdmin. When the task is executed in the backend, this zipped file is extracted and used.
 
+Usage
+=====
+Here is an example of how to use the ``pyflyte run`` command. The following code defines a task and a workflow that calls the task.
+
+.. code-block:: python
+
+    @task
+    def my_task(input1: int, input2: int) -> int:
+        return input1 + input2
+    @workflow
+    def my_workflow(input1: int, input2: int) -> int:
+        return my_task(input1=input1, input2=input2)
```

Comment:
I think you need a blank line here

Reply:
@neverett Really appreciate your help! This is my first time writing rst files, so I am wondering if there's anything useful for finding syntax errors in rst files? Thanks~

Reply:
No problem! We should probably incorporate a rst linter into the pre commit checks to make these errors more visible, but for now, you can check the logs by clicking "Details" next to the Monodocs Build check and searching the logs for the string "Warning treated as error". (If the log file is big to display, you may have to download it and search locally -- you can do that by clicking the gear icon in the upper right corner on the build details page.)